tests/ResponseTest.php

```diff
@@ -74,6 +88,13 @@ public function basicResponsesDataProvider(): array
             ['success' => true],
           ],
 
+          'respondWithSuccess(), null response data' => [
+            'respondWithSuccess',
+            [null],
+            Response::HTTP_OK,
+            ['success' => true],
+          ],
```

Comment:
This case was missing and failed before my changes. Null is allowed based on phpdocs and type hints. There are few similar cases down below.